server/tests/test_errors.py

```diff
@@ -140,6 +140,7 @@ def test_bgcompute_batch_tallies_errors(
 ):
     election = Election.query.get(election_id)
     election.audit_type = AuditType.BATCH_COMPARISON
+    election.ballot_polling_type = BallotPollingType.BRAVO
```

Comment:
seems like `election.ballot_polling_type` shouldn't be set in a batch comparison audit